bin/git-authors

```diff
@@ -52,5 +51,4 @@ if $LIST; then
   authors
 else
   authors >> "$FILE"
-  test -n "$EDITOR" && $EDITOR "$FILE"
```

Comment:
In the future, please do not mix changes that affect behavior and the testing suite in the same PR. Besides the other issues [I mentioned](https://github.com/tj/git-extras/pull/1098#discussion_r1377597263), doing so will mean that the change in behavior is not properly documented in the release notes.

Reply:
To ensure this breaking change will show up in the release notes, you can rebase your changes into two commits:
1. One for this breaking change and associated documentation, etc. changes
2. Two for the additional unit test and other things
Then, this PR can be "rebased and merged" against `main`, rather than "squshed and merged" as is usually in this repository. That is my recommendation, but I will delegate the decision an and enforcement to @spacewander

Reply:
> For the future, please do not mix changes that affect behavior and the testing suite in the same PR. Besides the other issues [I mentioned](https://github.com/tj/git-extras/pull/1098#discussion_r1377597263), doing so will mean that the change in behavior is not properly documented in the release notes. Generally, as a user of software, this is frustrating, and I do not wish to perpetuate this.

Copy that and I agree this.

Reply:
Yes. We can split the break change out and give it a more obvious name